llvm/lib/Support/AutoConvert.cpp

```diff
@@ -11,30 +11,37 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifdef __MVS__
```

Comment:
See my previous comment.  If we write the interface in AutoConvert.h so the functions are inline functions that do nothing for non-z/OS and then call these functions for z/OS, we can keep this file are z/OS only and not have function calls that return right away.

Reply:
Thanks, I've implemented this solution now